### PR TITLE
OTA support for gta4xl and gta4xlwifi

### PR DESCRIPTION
**Pull request by PHATWalrus**

### CODE REVIEW

File: gta4xl.json [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+                        "sha256": "17af3fa7aaf8e8b93cfb4285ab1e2450f4080fefd1e3a7035a65a51c230519ff",
+                        "size": 987128566,
+                        "version": "9.7",
+                        "buildtype": "Monthly",
+                        "forum": "https://forum.xda-developers.com/t/rom-unofficial-gta4xl-gta4xlwifi-13-0-crdroidandroid-v9-7-tab-s6-lite.4610281/",
+                        "gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/Addons-T/",
+                        "firmware": "https://lineage.linux4.de/fw_update/gta4xl.html",
+                        "modem": "",
+                        "bootloader": "",
+                        "recovery": "/gta4xl/crdroid/recovery.img",
```
Comment (gwolf2u) [inline gta4xl.json]: needs to be http(s) url

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: gta4xlwifi.json [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-			"kernel": "https://github.com/crdroidandroid/android_kernel_samsung_gta4xl"
-		}
-	]
+        "response": [
+                {
+                        "maintainer": "PhatWalrus",
+                        "oem": "Samsung",
+                        "device": "Galaxy Tab S6 Lite Wifi",
+                        "filename": "crDroidAndroid-13.0-20230727-gta4xlwifi-v9.7.zip",
+                        "download": "https://forum.xda-developers.com/t/rom-unofficial-gta4xl-gta4xlwifi-13-0-crdroidandroid-v9-7-tab-s6-lite.4610281/",
```
Comment (gwolf2u) [inline gta4xlwifi.json]: this should be generated download url not forum

File: gta4xlwifi.json [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+                        "timestamp": 1690437255,
+                        "md5": "31cbca8e0b2c56a87ef748604786a452",
+                        "sha256": "3e6d7be178b45830bb52b121c97b1671dd2057e77414684dc21f996b64abb17a",
+                        "size": 960931131,
+                        "version": "9.7",
+                        "buildtype": "Monthly",
+                        "forum": "https://forum.xda-developers.com/t/rom-official-gta4xl-gta4xlwifi-13-0-crdroidandroid-v9-x.4520949/",
```
Comment (gwolf2u) [inline gta4xlwifi.json]: update forum url to proper one

File: gta4xl.json [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
+                        "sha256": "17af3fa7aaf8e8b93cfb4285ab1e2450f4080fefd1e3a7035a65a51c230519ff",
+                        "size": 987128566,
+                        "version": "9.7",
+                        "buildtype": "Monthly",
+                        "forum": "https://forum.xda-developers.com/t/rom-unofficial-gta4xl-gta4xlwifi-13-0-crdroidandroid-v9-7-tab-s6-lite.4610281/",
+                        "gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/Addons-T/",
+                        "firmware": "https://lineage.linux4.de/fw_update/gta4xl.html",
+                        "modem": "",
+                        "bootloader": "",
+                        "recovery": "/https://sourceforge.net/projects/s6lite/files/gta4xl/crdroid/recovery.img/download",
```
Comment (gwolf2u) [inline gta4xl.json]: remove leading /

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: gta4xl.json (hunk 4 above)
Comment (PHATWalrus) [reply gta4xl.json]: Wdym?

Comment (gwolf2u) [reply gta4xl.json]: image

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: gta4xlwifi.json [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
-			"kernel": "https://github.com/crdroidandroid/android_kernel_samsung_gta4xl"
-		}
-	]
+        "response": [
+                {
+                        "maintainer": "PhatWalrus",
+                        "oem": "Samsung",
+                        "device": "Galaxy Tab S6 Lite Wifi",
+                        "filename": "crDroidAndroid-13.0-20230727-gta4xlwifi-v9.7.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/gta4xl/9.x/crDroidAndroid-13.0-20230727-gta4xlwifi-v9.7.zip/download",
```
Comment (gwolf2u) [inline gta4xlwifi.json]: bad dl url
gta4xlwifi

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: gta4xlwifi.json (hunk 5 above)
Comment (PHATWalrus) [reply gta4xlwifi.json]: Can u kill me ?

Comment (gwolf2u) [reply gta4xlwifi.json]: probably :))

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**